bootstrap-app/src/cmr/bootstrap/api/bulk_index.clj

```diff
@@ -159,6 +159,25 @@
      {:status 202
       :body {:message (msg/index-subscriptions params provider-id result)}})))
 
+(defn index-generics
+  "(Re-)Index the generic documents of a given type stored in metadata-db. If a provider-id is passed,
+  only the documents for that provider will be indexed. With no provider-id,
+  all providers' documents are (re-)indexed.
```

Comment:
I think this is supposed to be provider's

Reply:
provider's = belonging to a provider
providers' = belonging to multiple providers

In this case it should be the latter, because if no provider-id is specified, it indexes the documents of all providers.